README.md

```diff
@@ -428,6 +428,40 @@ memory, use:
 * `export NCCL_P2P_DISABLE=1` for NCCL.
 * `--mca btl_smcuda_use_cuda_ipc 0` flag for OpenMPI and similar flags for other vendors.
 
+## Analyzing Horovod Performance
+
+Horovod has an ability to record the timeline of its activity, called Horovod Timeline.
+
+![Horovod Timeline](https://user-images.githubusercontent.com/16640218/29540325-a80e2140-8682-11e7-9f21-ada1613948be.png)
+
+In order to record Horovod Timeline, please set `HOROVOD_TIMELINE` environment variable to location of the timeline
+file to be created.  This file is only recorded on rank 0, but it contains information about activity of all workers.
+
+```bash
+$ HOROVOD_TIMELINE=/path/to/timeline.json mpirun -np 4 -x HOROVOD_TIMELINE python train.py
+```
+
+You can then open timeline file using [chrome://tracing](chrome://tracing) facility of [Chrome](https://www.google.com/chrome/browser/) browser.
```

Comment:
open **the** timeline file
using **the** [chrome://tracing](chrome://tracing) facility of **the** [Chrome](https://www.google.com/chrome/browser/) browser